NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -220,17 +228,9 @@ def begin(self,
         self._adc_ref_voltage_range = 2.5 * units.volt
         self._adc_n_bits = 12
         
-        self.__max_trigger_rate = max_trigger_rate
-        self.__run_types = run_types
-        
-        if run_time_range is not None:
-            convert_time = lambda t: None if t is None else astropy.time.Time(t)
-            self._time_low = convert_time(run_time_range[0])
-            self._time_high = convert_time(run_time_range[1])
-        else:
-            self._time_low = None
-            self._time_high = None            
-        
+        self._overwrite_sampling_rate = overwrite_sampling_rate
+            
+        # Initialize run table for run selection
         self.__run_table = None
```

Comment:
If that is a use case we should do it. Should we also add the interface set_selectors?

Reply:
Check it out. We can also think about a similar interface for the run select options and pass to begin() only a dict. This would simplify the doc string because the discription of the run selection option with move to set_run_selection or similar. What do you think?

Reply:
Yup, this looks nice, thanks. I guess for my purposes I don't necessarily need a separate `set_run_selection` option, and I guess there is a trade-off between simplifying the begin function and 'crowding' the class with methods. I'll leave this up to your/others' judgement.